apps/storefront/vite.config.ts

```diff
@@ -3,18 +3,14 @@ import legacy from '@vitejs/plugin-legacy';
 import react from '@vitejs/plugin-react';
 import path from 'path';
 import { visualizer } from 'rollup-plugin-visualizer';
-import { defineConfig, loadEnv } from 'vite';
+import { defineConfig, loadEnv, UserConfig } from 'vite';
+import { ViteUserConfig } from 'vitest/config';
 
-export default defineConfig(({ mode }) => {
+export default defineConfig(({ mode }): UserConfig & Pick<ViteUserConfig, 'test'> => {
```

Comment:
Added types here for autocomplete. `defineConfig` does not include the `test` property in the config.